migrations/2023-03-15-185959_add_dispute_table/up.sql

```diff
@@ -23,7 +23,7 @@ CREATE TABLE dispute (
     modified_at TIMESTAMP NOT NULL DEFAULT now()::TIMESTAMP
 );
 
-CREATE UNIQUE INDEX dispute_id_index ON dispute (dispute_id);
+CREATE UNIQUE INDEX merchant_id_dispute_id_index ON dispute (merchant_id, dispute_id);
```

Comment:
Check whether a change is required in `down.sql` as well.

Reply:
We are dropping `dispute` table in down.sql, so are indices!